cmd/kava/cmd/app.go

```diff
@@ -110,9 +111,22 @@ func (ac appCreator) appExport(
 	appOpts servertypes.AppOptions,
 ) (servertypes.ExportedApp, error) {
 
-	panic("TODO") // TODO
+	homePath, ok := appOpts.Get(flags.FlagHome).(string)
+	if !ok || homePath == "" {
+		return servertypes.ExportedApp{}, errors.New("application home not set")
+	}
+
+	var tempApp *app.App
+	if height != -1 {
+		tempApp = app.NewApp(logger, db, homePath, traceStore, ac.encodingConfig, app.Options{SkipLoadLatest: true, InvariantCheckPeriod: uint(1)})
```

Comment:
Do we intentionally hardcode `InvariantCheckPeriod` here?  We could use the `server.FlagInvCheckPeriod` flag -- unsure it it matters for `LoadHeight` & `ExportAppStateAndValidators`